### PR TITLE
Updated Dockerfile

### PR DESCRIPTION
**Pull request by mayanksharma3**
### Clubhouse Ticket
[ch113]({https://app.clubhouse.io/manim-dsl/story/113/update-dockerfile-to-use-official-manim})

### Description

Updated Dockerfile

### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue) 

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
-# Install manim (community)
-RUN pip3 install manimce
+# Install python3 + all dependancies for manim
+RUN apt-get update && apt-get install -y python3 python3-pip sox ffmpeg libcairo2 libcairo2-dev
+
+# Update pip for opencv dependancy
+RUN pip3 install --upgrade pip
+
+# Install manim
+RUN pip3 install manimlib
```
[the remark this answers is not in the document]
Comment (mayanksharma3) [reply Dockerfile:13]: Yes, it's not manimce anymore

[the remark this answers is not in the document]
Comment (arj119) [reply Dockerfile:13]: If we were to switch to manimce would it work if we changed the top line to from manim import *

[the remark this answers is not in the document]
Comment (mayanksharma3) [reply Dockerfile:13]: Some things like TextMobject don't exist in one, so not right now I believe, let's just stick to the current working version

**Review by arj119 — APPROVED** [review]
LGTM 👍

**Pull request merged**